include/mapbox/variant.hpp

```diff
@@ -84,25 +84,27 @@ struct MAPBOX_VARIANT_DEPRECATED static_visitor
     ~static_visitor() {}
 };
 
+using type_index_t = unsigned int;
```

Comment:
On x86/x64 32-bit `int` may grant performance advantages over integers of another size (including `char`).